src/pages/code-of-conduct.js

```diff
@@ -9,176 +10,197 @@ import Ul, { Li } from '../components/layout/Ul'
+                    disruption of talks or other events, inappropriate physical
+                    contact, and unwelcome sexual attention.
+                  </P>
+                  <P>
+                    Participants asked to stop any harassing behavior are
+                    expected to comply immediately.
+                  </P>
+                  <P>
+                    Sponsors and coaches are also subject to the anti-harassment
+                    policy. In particular, sponsors and coaches should not use
+                    sexualised images, activities, or other material.
+                    Additionally, sponsors and coaches should not use sexualised
+                    clothing/uniforms/costumes, or otherwise create a sexualised
+                    environment.
+                  </P>
+                  <P>
+                    If a participant engages in harassing behavior, the event
+                    organisers may take any action they deem appropriate,
+                    including warning the offender or expulsion from the event
+                    with no refund.
+                  </P>
+                  <P>
+                    If you are being harassed, notice that someone else is being
+                    harassed, or have any other concerns, please contact a
+                    member of React GraphQL Academy staff immediately. Our
+                    coaches and representatives will identify themselves to you
+                    at the start of any event and may also be identified via
+                    branded clothing.
+                  </P>
+                  <P>
+                    React GraphQL Academy staff will be happy to help
+                    participants contact local law enforcement, source escorts,
+                    or otherwise assist those experiencing harassment to feel
+                    safe for the duration of the training event. We value your
+                    attendance immensly and want you to have the best experience
+                    possible.
+                  </P>
+                  <P>
+                    We expect participants to follow these rules at our
+                    trainings and training-related social events.
+                  </P>
+                </Col>
+                <Col md={5} mdOffset={1}>
+                  <Image src={moreDetailImgSrc} />
+                </Col>
+              </Row>
+            </Grid>
+          </Section>
+          <UpcomingTrainingSection trainings={trainings} />
+        </React.Fragment>
+      )
+    }}
   </Layout>
 )
 
+export const query = graphql`
+  query condeOfConductImgs {
+    points: file(absolutePath: { regex: "/codeofconduct_keypoints/" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    detail: file(absolutePath: { regex: "/codeofconduct_moredetail/" }) {
+      childImageSharp {
+        fluid {
```

Comment:
@ric9176 why don't we use a maxWidth so we make sure it's not too big?. Example:
- https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/master/src/components/layout/Header.js#L279
- https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/master/src/components/layout/Header.js#L266